### ISSUE TITLE
TypeError when trying to create new file (spc f n)

### ISSUE THREAD

**Issue by mtsitzer** [VSpaceCode/VSpaceCode#103]
**Describe the bug**
I am getting a `TypeError: Cannot read property 'isReadonly' of undefined` when trying to create a new file (spc f n).

I've installed this extension on two devices and have this issue on both.

**To Reproduce**

1. Normal mode in a file
2. Type spacebar, this opens up Vspacecode menu, then f to go to file section, the n for new file
3. Get error

**Expected behavior**
A new untitled editor tab would open up

**Screenshots**
![image](https://user-images.githubusercontent.com/28517224/93654208-2d32b380-f9d1-11ea-8ce9-f6e3574ffbbb.png)

**Additional context**

**System information**

![image](https://user-images.githubusercontent.com/28517224/93654516-bc8c9680-f9d2-11ea-8ca3-61a9b611c5f2.png)

![image](https://user-images.githubusercontent.com/28517224/93654527-d201c080-f9d2-11ea-9b46-469921634b9e.png)

![image](https://user-images.githubusercontent.com/28517224/93654529-db8b2880-f9d2-11ea-9d44-90bcdc4f11d6.png)

**Comment by stevenguh** [comment]
Good catch. This only happens when there is no workspace (no folders are opened). To fix this, we can either implement a custom command, or find another command that works when there is no workspace.

**Comment by stevenguh** [comment]
Technically, you can't create a named file with the vscode explorer if no folder is opened, but you can create untitled/titled buffer in that case. There are a few solutions in mind, they all involved implementation of a custom command:
1. When no folder is opened, a new untitled buffer is created for `spc f b`
2. When no folder is opened, display a better error message for `spc f b`
3. When no folder is opened, an input is prompt is shown to ask for the name of the new buffer (This is similar to 4, except the prompt will only shown if no folder is opened)
4. Implement something similar to `:e` option in VSCode Vim, and still has some corner cases for VSCode Remote

I am hesitant with option 4, since vscode's api is kind limited to handle file and URI for all cases, see https://github.com/microsoft/vscode/issues/105969
